### ISSUE TITLE
Community logo

### ISSUE THREAD

**Issue by ROCKNROLLKID** [ROCKNROLLKID/Open-Source-Community#1]
![community logo](https://cloud.githubusercontent.com/assets/6929591/10747705/48c41b30-7c26-11e5-947e-b8d1e78d3b9c.png)

Here you can edit and resubmit the community logo. All credit will be rightly given to each user who helps. If you plan on redistributing our logo, you must put somewhere that the logo came from and was designed from by the Open-Source Community. Please ask if you want modify it and redistribute it.

**Comment by testman42** [comment]
Would it be wrong to use Open Source Initiative logo for group? It's way more well known and representative of open source movement / community.

**Comment by ROCKNROLLKID** [comment]
I did have plans to make a copyright to the logo. If you do use our logo, you must put somewhere that the logo came from and was designed from by the Open-Source Community.

However, you can still edit it, and resubmit the logo to us, as you wish.

**Comment by kriswema** [comment]
@ROCKNROLLKID You might want to put it under a [Creative Commons](https://creativecommons.org/licenses/) license then. Maybe it would also make sense to host an SVG version of the logo in the repository.

**Comment by ROCKNROLLKID** [comment]
I should have put the entire group under the Creative Common license, including the logo. Is there anyway I can edit the license or would I have to make a new GitHub page (still a noob to GitHub)?

**Comment by kriswema** [comment]
@ROCKNROLLKID Alright. You create a new file in the main repository view and give it the name LICENSE. The license template picker with a quite extensive list of licenses should now appear at the right side.

Check out http://choosealicense.com/. I've always been a fan of Creative Commons, but I guess the MIT license would work just as well.

**Comment by ROCKNROLLKID** [comment]
Done. The group is now under the Creative Common license. This will also apply to the logo, too, right or do I need to put some kind of copyright on the logo?

**Comment by ROCKNROLLKID** [comment]
@testman42 I would like to know what group you are planning on using our logo for? Would you mind linking it to us.

**Comment by AdmiringWorm** [comment]
@ROCKNROLLKID may I suggest putting the icon in the repository itself so it is covered by the [CC] license, and possible put the information in the [README](https://github.com/ROCKNROLLKID/Open-Source-Community/blob/master/README.md) file?

**Comment by testman42** [comment]
@ROCKNROLLKID I believe my question my misread / misunderstood.
I was asking why does Steam group named Open Source Community use custom logo instead of well known and recognised Open Source Initiative logo:

![OSI Logo](https://upload.wikimedia.org/wikipedia/commons/4/4e/Open_Source_Initiative_keyhole.svg)

**Comment by ROCKNROLLKID** [comment]
@AdmiringWorm As far as I am concerned and by reading how GitHub works and all, there isn't really a way to put it in the main respo. I also heard that putting it here was the best/easiest way.

@testman42 I thought making our own logo would be better. Someone offered to do so, so I decided we would make our own.

**Comment by AdmiringWorm** [comment]
true, you can't add the icon to the main repos through the website.
However the way Github works is that you're using a program called _git_ to make a clone of this repository on your local machine.
Then you make the changes or create the files in that local _clone_,
add them to the change tracker (in this case git) and commit the changes.
Then when all that is done you push those changes to the remote repository.

To learn more about git and see how to set it up you could check out the github help pages: https://help.github.com/

Also putting the logo here is sure the easiest, but in my opinion not the best.
The best would be _(still in my opinion)_ to add it in the main repo and but the information on how
it is to be used by third-party developers.
One way it would be better is it would automatically be covered by the License file in the repository (all files in the repository is covered by it),
Another thing that would be a plus is that other developers could then use a CDN called https://rawgit.com/ to put a link to the logo instead of copying to their projects.
It would also be noticed by others more easily, not everyone checks the issues unless the they actually have an issue or a suggestion.

**Comment by ROCKNROLLKID** [comment]
I think, for now, I will keep it how it is. Thank you for your suggestion. I may consider it in the future, though.
